reqhandler/fields.go

```diff
@@ -0,0 +1,33 @@
+package reqhandler
+
+type FieldName string
+
+const (
+	ID       FieldName = "id"
+	QUERY              = "query"
+	OPCODE             = "opcode"
+	AUTHANS            = "authans"
+	TRUNC              = "trunc"
+	RDESC              = "rdesc"
+	RAVAIL             = "ravail"
+	RESPCODE           = "respcode"
+	QDCOUNT            = "qdcount"
+	ANCOUNT            = "ancount"
+	NSCOUNT            = "nscount"
+	ARCOUNT            = "arcount"
+)
```

Comment:
Can we use the names for the header fields here: [https://tools.ietf.org/html/rfc1035#section-4.1.1](https://tools.ietf.org/html/rfc1035#section-4.1.1)? So AA isntead of AUTHANS and so on because that makes it easier to map to the diagram in the RFC.